capture-sdk/sdk/src/main/java/net/gini/android/capture/analysis/AnalysisActivity.java

```diff
@@ -46,18 +46,19 @@
  * document and an activity indicator while the document is being analyzed by the Gini API.
  *
  * <p> <b>Note:</b> The title from the ActionBar was removed. Use the activity indicator message
- * instead by overriding the string resource named {@code gc_analysis_activity_indicator_message}. The message is displayed for images
- * only.
+ * instead by overriding the string resource named {@code gc_analysis_activity_indicator_message}.
```

Comment:
[Request] You can replace this whole javadoc for AnalysisActivity with `Internal use only`. It's not part of the public API anymore.

Sorry that I wasn't able to tell you that earlier. 🙈